packages/message-encryption/src/ecies.spec.ts

```diff
@@ -6,6 +6,7 @@ import { getPublicKey } from "./crypto/index.js";
 import { createDecoder, createEncoder } from "./ecies.js";
 
 describe("Ecies Encryption", function () {
+  this.timeout(20000);
```

Comment:
Why?

Reply:
With this PR, since I had to replace the mocha config in all packages, I activated parallel runs in the CI for all packages. 
And while parallel runs, test are taking longer to execute.
I saw that tests failing in the CI with the test timing out with the default 2000 ms timeout so I've increased it similar with other tests.

Reply:
Are these timeouts related to `nwaku-jswaku` communication or it's random?

Reply:
I believe that it's related to either how mocha implements concurrency or by the fact that because we run multiple containers at the same time when running tests in parallel, things start to slow down.
Github actions machine is also pretty low on resources so it's kinda expected to see such timeouts when running tests in parallel.
Those tests run in less than 1 second in serial mode that's why I don't think it's related to waku at all